### ISSUE TITLE
Command line: Support systems with .NET 5 installed

### ISSUE THREAD

**Issue by GrahamTheCoder** [icsharpcode/CodeConverter#674]
The tests don't run now that I've installed .NET 5. I expect the command line will have issues too:
```csharp
Fix compilation errors before conversion for an accurate conversion, or as a last resort, use the best effort conversion option:

14 workspace errors:
Msbuild failed when processing the file 'C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\WindowsAppVb\WindowsAppVb.vbproj' with message: C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\VbNetStandardLib\VbNetStandardLib.vbproj: (0, 0): The SDK resolver "Microsoft.DotNet.MSBuildSdkResolver" failed to run. Method not found: 'Microsoft.Build.Framework.SdkResult Microsoft.Build.Framework.SdkResultFactory.IndicateSuccess(System.String, System.String, System.Collections.Generic.IDictionary`2<System.String,System.String>, System.Collections.Generic.IDictionary`2<System.String,Microsoft.Build.Framework.SdkResultItem>, System.Collections.Generic.IEnumerable`1<System.String>)'.
Msbuild failed when processing the file 'C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\WindowsAppVb\WindowsAppVb.vbproj' with message: C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\VbNetStandardLib\VbNetStandardLib.vbproj: (0, 0): The SDK 'Microsoft.NET.Sdk' specified could not be found.
Msbuild failed when processing the file 'C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\WindowsAppVb\WindowsAppVb.vbproj' with message: C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets: (2161, 9): The "UseResolveAssemblyReferenceService" parameter is not supported by the "ResolveAssemblyReference" task. Verify the parameter exists on the task, and it is a settable public instance property.
Msbuild failed when processing the file 'C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\WindowsAppVb\WindowsAppVb.vbproj' with message: C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets: (2121, 5): The "ResolveAssemblyReference" task could not be initialized with its input parameters. 
Msbuild failed when processing the file 'C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\VbNetStandardLib\VbNetStandardLib.vbproj' with message: The SDK 'Microsoft.NET.Sdk' specified could not be found.  C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\VbNetStandardLib\VbNetStandardLib.vbproj
Msbuild failed when processing the file 'C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\ConsoleApp2\CSharpConsoleApp.csproj' with message: C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\CSharpNetStandardLib\CSharpNetStandardLib.csproj: (0, 0): The SDK resolver "Microsoft.DotNet.MSBuildSdkResolver" failed to run. Method not found: 'Microsoft.Build.Framework.SdkResult Microsoft.Build.Framework.SdkResultFactory.IndicateSuccess(System.String, System.String, System.Collections.Generic.IDictionary`2<System.String,System.String>, System.Collections.Generic.IDictionary`2<System.String,Microsoft.Build.Framework.SdkResultItem>, System.Collections.Generic.IEnumerable`1<System.String>)'.
Msbuild failed when processing the file 'C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\ConsoleApp2\CSharpConsoleApp.csproj' with message: C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\CSharpNetStandardLib\CSharpNetStandardLib.csproj: (0, 0): The SDK 'Microsoft.NET.Sdk' specified could not be found.
Msbuild failed when processing the file 'C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\ConsoleApp2\CSharpConsoleApp.csproj' with message: C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets: (2161, 9): The "UseResolveAssemblyReferenceService" parameter is not supported by the "ResolveAssemblyReference" task. Verify the parameter exists on the task, and it is a settable public instance property.
Msbuild failed when processing the file 'C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\ConsoleApp2\CSharpConsoleApp.csproj' with message: C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets: (2121, 5): The "ResolveAssemblyReference" task could not be initialized with its input parameters. 
Msbuild failed when processing the file 'C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\CSharpNetStandardLib\CSharpNetStandardLib.csproj' with message: The SDK 'Microsoft.NET.Sdk' specified could not be found.  C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\CSharpNetStandardLib\CSharpNetStandardLib.csproj
Msbuild failed when processing the file 'C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\NetCore\NetCore.csproj' with message: The SDK 'Microsoft.NET.Sdk' specified could not be found.  C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\NetCore\NetCore.csproj
Msbuild failed when processing the file 'C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\VbLibrary\VbLibrary.vbproj' with message: C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets: (2161, 9): The "UseResolveAssemblyReferenceService" parameter is not supported by the "ResolveAssemblyReference" task. Verify the parameter exists on the task, and it is a settable public instance property.
Msbuild failed when processing the file 'C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\VbLibrary\VbLibrary.vbproj' with message: C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets: (2121, 5): The "ResolveAssemblyReference" task could not be initialized with its input parameters. 
Msbuild failed when processing the file 'C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\ConsoleApp4\ConsoleApp4.vbproj' with message: The SDK 'Microsoft.NET.Sdk' specified could not be found.  C:\Users\Graham\Documents\GitHub\CodeConverter\Tests\TestData\MultiFileCharacterization\SourceFiles\ConsoleApp4\ConsoleApp4.vbproj
```

**Comment by BrianFreemanAtlanta** [comment]
I've been looking into this. It appears to be isolated to the test fixture. I've tried out the command line and it works without a problem. I've determined that it's a reference to `Microsoft.Build` and `Microsoft.Build.Task.Core` that is causing the problem. In the package references for there is the following item group.  
``` 
<ItemGroup Label="ReSharper test runner requirements">
    <PackageReference Include="Microsoft.CodeAnalysis.Features" Version="3.4.0" />
    <PackageReference Include="Microsoft.Build" Version="16.4.0" />
    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="16.4.0" />
</ItemGroup>
```

With the release of .NET 5.0 there is an update to both of the build packages to 16.8.0. Updating the package references allows the tests to run successfully. I can do a pull request with the updated references tomorrow.
